spec.md

```diff
@@ -1003,10 +1003,22 @@ For success responses, the following fields are defined:
 
 | Response Field | Type | Description |
 | --- | --- | --- |
+| dashboard_url | string | The updated URL of a web-based management user interface for the Service Instance; we refer to this as a service dashboard. The URL MUST contain enough information for the dashboard to identify the resource being accessed (`0129d920a083838` in the example below). Note: a Service Broker that wishes to return `dashboard_url` for a Service Instance MUST return it with the initial response to the update request, even if the service is updated asynchronously. If present, MUST be a non-empty string. |
```

Comment:
Is this really the updated URL?

Reply:
I'm not following your question

Reply:
I assume he means that it's not necessarily updated? It could be the original one?

Reply:
That's an impl choice I guess - we don't mandate the value sent.